package.json

```diff
@@ -8,7 +8,8 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint $(git ls-files '*.js')",
-    "test": "mocha --use_strict --check-leaks --no-exit "
+    "test": "mocha --use_strict --check-leaks --no-exit ",
+    "coverage": "nyc --check-coverage --exclude 'tests/*' --lines=90 --reporter=text mocha --use_strict --check-leaks --no-exit "
```

Comment:
How does nyc fare for coverage ? We've been historically using istanbul for this in S3 projects.
Simple curiosity.

Reply:
<confused mode on> Aren't they the same? From the github repo, nyc is the cli to Istanbul (https://github.com/istanbuljs/nyc). </confused off>